lab01/lab01.py

```diff
@@ -11,6 +11,13 @@ def falling(n, k):
     1
     """
     "*** YOUR CODE HERE ***"
+    i=1
```

Comment:
Better variable naming. If here `i` means the return value for this function, better to name it `result`

Reply:
turn I into total